types/ir/ir.go

```diff
@@ -113,8 +125,8 @@ type Service struct {
 	Replicas                    int
 	Networks                    []string
 	OnlyIngress                 bool
-	Daemon                      bool //Gets converted to DaemonSet
-	StatefulSet                 bool // Whether the service gets converted to a StatefulSet
+	Daemon                      bool           //Gets converted to DaemonSet
+	DeploymentType              DeploymentType // Whether the service gets converted to a StatefulSet
```

Comment:
Can we change the comment as it's not just for `StatefulSet`?

Reply:
Resolved in [1608c10](https://github.com/konveyor/move2kube/pull/1098/commits/1608c1008a61017cf59810042d9021b06b034ae7)